addons/website_hr_recruitment/views/website_hr_recruitment_templates.xml

```diff
@@ -851,22 +851,22 @@
                 <a t-attf-href="/jobs?all_employment_types=1#{non_employment_type_params}"
                     t-attf-class="dropdown-item d-flex align-items-center justify-content-between nav-link#{'' if contract_type_id or is_untyped else ' active'}">
                     All Types
-                    <span t-attf-class="badge rounded-pill#{' bg-primary' if contract_type_id else ' bg-light text-primary'} ms-2" t-out="count_per_employment_type.get('all', '0')"/>
+                    <span t-attf-class="badge rounded-pill text-bg-primary ms-2" t-out="count_per_employment_type.get('all', '0')"/>
```

Comment:
Same question as in other places.. no need for differentiation?

Reply:
Answered [here](https://github.com/odoo/odoo/pull/162935#discussion_r1706970291)